src/charm.py

```diff
@@ -109,6 +113,36 @@ def __init__(self, *args):
             self._certificates.on.certificate_expiring, self._on_certificate_expiring
         )
 
+    def _on_install(self, event: InstallEvent) -> None:
+        client = Client()
+        logger.info("Creating external AMF service")
```

Comment:
(Applicable to both logging)

I think we generally place the logging messages after the action was completed. Ex. `created external AMF service` instead of `creating external AMF service`.

Reply:
Done